filecoin-proofs/src/lib.rs

```diff
@@ -7,6 +7,8 @@ extern crate failure;
 #[macro_use]
 extern crate log;
 
+extern crate bincode;
```

Comment:
no need for this

Reply:
Hrm, ok, the serialize/deserialize didn't want to work without it.  Maybe it's not being done in the best way?

Reply:
strange